src/plugins/discourse/createGraph.test.js

```diff
@@ -52,20 +50,26 @@ describe("plugins/discourse/createGraph", () => {
       return this._posts;
     }
     users(): $ReadOnlyArray<User> {
-      const users = [];
-      const userNames = new Set();
+      const usernames = new Set();
+      const trustLevels: Map<string, number> = new Map();
 
-      for (const {authorUsername} of this.posts()) {
-        userNames.add(authorUsername);
-        users.push({username: authorUsername, trustLevel: 3});
+      for (const {authorUsername, trustLevel} of this.posts()) {
+        usernames.add(authorUsername);
+        trustLevels.set(authorUsername, trustLevel);
       }
       for (const {authorUsername} of this.topics()) {
-        if (!userNames.has(authorUsername)) {
-          userNames.add(authorUsername);
-          users.push({username: authorUsername, trustLevel: 3});
-        }
+        usernames.add(authorUsername);
+      }
+      for (const {username} of this.likes()) {
+        usernames.add(username);
```

Comment:
@befitsandpiper this is a change I was suggesting on Discord--we now "realistically" choose the user trust level based on the underlying post data, rather than inventing it arbitrarily

Reply:
I see, yeah that makes sense. Sorry about that.